docs/dev-guide/3.0-development/rest-planning.rst

```diff
@@ -0,0 +1,8 @@
+.. warning::
+    All documents within the 3.0-Development section should be considered temporary.
+
+
+REST API planning
+=====================
```

Comment:
4 less `=` to match length of section name  but *really* don't know if it's important.